Packs/CheckpointFirewall/Playbooks/Checkpoint_-_Publish_configuration.yml

```diff
@@ -0,0 +1,409 @@
+            value:
+              complex:
+                root: CheckPoint
+                accessor: show-task.status
+            iscontext: true
+          right:
+            value:
+              simple: succeeded
+    view: |-
+      {
+        "position": {
+          "x": -680,
+          "y": 590
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "16":
+    id: "16"
+    taskid: 0ae1577b-3967-4d80-8286-ea5fc8aa195d
+    type: regular
+    task:
+      id: 0ae1577b-3967-4d80-8286-ea5fc8aa195d
+      version: -1
+      name: Print Publish Error
+      description: Prints an error entry with a given message.
+      scriptName: PrintErrorEntry
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "17"
+    scriptarguments:
+      message:
+        simple: Checkpoint Publish Failed
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -400,
+          "y": 770
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "17":
+    id: "17"
+    taskid: dcfa9a18-2748-4e6d-81f7-5e946c0e325c
+    type: regular
+    task:
+      id: dcfa9a18-2748-4e6d-81f7-5e946c0e325c
+      version: -1
+      name: Fix Error and publish manually
+      description: 'User Manually add resolve'.
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "9"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -400,
+          "y": 940
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "24":
+    id: "24"
+    taskid: 02b3925d-6ca8-494d-8369-4087983df5af
+    type: condition
+    task:
+      id: 02b3925d-6ca8-494d-8369-4087983df5af
+      version: -1
+      name: Filter TaskID
+      description: Filter TaskID to validate only new push configuration TaskID.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "9"
+      "yes":
+      - "13"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: CheckPoint
+                accessor: publish.task-id
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": -200,
+          "y": 10
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 2425,
+        "width": 1180,
+        "x": -680,
+        "y": -560
+      }
+    }
+  }
+inputs:
+- key: sid
+  value: {}
+  required: true
+  description: SID - Session unique identifier as it returned by the login request
+    required for publish changes. done by this SID will be seen by all users only
+    after publish is called and finish.
+outputs: []
```

Comment:
Same as in the previous playbook, consider adding outputs

Reply:
migrated.